rust/osm2lanes/src/overpass.rs

```diff
@@ -0,0 +1,43 @@
+use serde::Deserialize;
+
+use crate::tags::Tags;
+
+#[derive(Debug, Clone, Deserialize)]
+struct OverpassResult {
+    // version: f32,
+    // osm3s: Osm3s
+    elements: Vec<Element>,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+struct Element {
+    r#type: ElementType,
```

Comment:
Whoaaaa is that how you use reserved words as field names? I never knew! (I've always done `#[serde(rename = "type")]` and named the field something else